lib/InjectorServer.js

```diff
@@ -15,7 +15,9 @@ function injectorServer(options) {
     // Get previously cached mirror if existed
     mirror = resolveCachedMirror(cache, mirror);
 
-    var serializer = makeMirrorSerializer(true);
+    var serializer = makeMirrorSerializer(true, {
+      maxStringLength: 10000
```

Comment:
We store this constant in two places at current time. (this is bad solution and I can rewrite it on something more complex, if you want)